src/Tribe/Ignored_Events.php

```diff
@@ -975,7 +977,32 @@ public function hook() {
 			 */
 			tribe_notice( 'legacy-ignored-events', array( $this, 'render_notice_legacy' ), 'dismiss=1&type=warning' );
 
+			add_filter( 'wp_count_posts', [ $this, 'patch_count_posts' ], 10, 3 );
+
 			return true;
 		}
+
+		/**
+		 * Patch post-list-table queries to include ignored status,
+		 * as WP just accesses the stati without checking if they exist.
+		 *
+		 * @see wp-admin/includes/class-wp-posts-list-table.php->get_views()
+		 *
+		 * @since TBD
+		 *
+		 * @param [type] $counts
+		 * @param [type] $type
+		 * @param [type] $perm
+		 * @return void
```

Comment:
:no_entry_sign: Function return type is void, but function contains return statement

<sub>hash: 860fb734d76be795bec86ae012c77636</sub>